project-backend/Controllers/JobController.cs

```diff
@@ -128,6 +129,24 @@ public IActionResult GetJobDetails([FromQuery] GetJobDetailsQueryObject job)
             }
 
         }
+
+        [HttpGet]
+        [Authorize]
+        [Route("Bids")]
+        [ProducesResponseType(typeof(GetJobBidsResponseObject), StatusCodes.Status200OK)]
```

Comment:
```suggestion
        [ProducesResponseType(typeof(GetJobBidsResponseObject), StatusCodes.Status200OK)]
        [ProducesResponseType(typeof(ValidationProblemDetails), StatusCodes.Status400BadRequest)]
```
This response type should be added because it can be returned to the client in case the query object fails validation

Reply:
If that's the case, should we add it to all the endpoints?

Reply:
Added